pipeline/beam_tables.py

```diff
@@ -459,11 +483,16 @@ def _unflatten_satellite(
   """
   if flattened_measurement:
     # Get common fields and update 'received' with array of all answer IPs
-    combined = flattened_measurement[0]
-    combined['received'] = [
-        answer['received'] for answer in flattened_measurement
-    ]
+    combined: Row = {'received': []}
+    for answer in flattened_measurement:
+      received = answer.pop('received', None)
+      combined.update(answer)
+      if received:
+        combined['received'].append(received)
     combined.pop('measurement_id')
+    # temp fix to remove extra vantage point tags
```

Comment:
is this when the vantage point and response are the same? If so add that info, if not can you explain more what the issue is here?